tests/github-action-helper.sh

```diff
@@ -74,13 +82,45 @@ deploy_with_custom_ns() {
   kubectl create -f "$3"
 }
 
+deploy_csi_driver_default_ns() {
+  kubectl create -f https://raw.githubusercontent.com/rook/rook/master/deploy/examples/filesystem-test.yaml
+  kubectl create -f https://raw.githubusercontent.com/rook/rook/master/deploy/examples/subvolumegroup.yaml
+  kubectl create -f https://raw.githubusercontent.com/rook/rook/master/deploy/examples/csi/rbd/storageclass-test.yaml
```

Comment:
why we are not creating cephfs storageclass here?

Reply:
it was missing earlier also, I added it now. Thanks